src/render.tsx

```diff
@@ -293,7 +293,7 @@ const renderHtml = (
   options: {
     collapse?: boolean;
     renderers?: HtmlRenderers;
-    style?: Style | (Style | undefined)[];
+    style?: Style | Style[];
```

Comment:
This signature matches react-pdf docs better, but maybe this was done for something else?